<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-NOPHRFF" failures="1" tests="1" name="Test" time="63.717" errors="0" timestamp="2021-06-23T00:14:49 EET">
  <testcase classname="testcases.VodafoneSearchQuery" name="InitiateVodafoneSearch" time="46.485">
    <failure type="java.lang.AssertionError" message="expected [10] but found [11]">
      <![CDATA[java.lang.AssertionError: expected [10] but found [11]
at testcases.VodafoneSearchQuery.InitiateVodafoneSearch(VodafoneSearchQuery.java:70)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- InitiateVodafoneSearch -->
</testsuite> <!-- Test -->
